components/data-table/index.jsx

```diff
@@ -240,6 +268,57 @@ class DataTable extends React.Component {
 		}
 	};
 
+	resizeFixedHeaders = () => {
+		if (this.props.onFixedHeaderResize) {
+			this.props.onFixedHeaderResize({
+				scrollerRef: this.scrollerRef,
+				theadRef: this.theadRef,
+			});
+		} else if (this.theadRef) {
+			let documentScrollLeft = 0;
+
+			if (document && document.documentElement) {
+				documentScrollLeft = document.documentElement.scrollLeft;
+			}
+
+			this.theadRef.querySelectorAll('th').forEach((column) => {
```

Comment:
`theadRefs` should probably be an array of DOM refs. `querySelectorAll` will work, but I would feel better about using the framework and pushing refs into an array on render and not querying the DOM. This code will also promotes creators of custom callbacks to query inside the component's DOM, too.

I'd recommend passing down a function to header sub component that will update the instance property in the grandparent (`index.jsx`) on render of the header.